supyr_struct/tag.py

```diff
@@ -476,7 +474,7 @@ def serialize(self, **kwargs):
             return data.serialize(**kwargs)
 
         temp = kwargs.pop('temp', True)
```

Comment:
If you are going to do this, then change it like this to ensure the buffer kwarg is removed if None
```python
        buffer = kwargs.pop('buffer', None)
        if buffer is not None:
            return data.serialize(buffer=buffer , **kwargs)

        temp = kwargs.pop('temp', True)
        backup = kwargs.pop('backup', True)
```

Reply:
Did that